provider/postgis/postgis.go

```diff
@@ -199,14 +212,54 @@ func NewTileProvider(config dict.Dicter) (provider.Tiler, error) {
 		return nil, err
 	}
 
+	auto := DefaultAuto
+	if len(layers) == 0 {
+		auto = true
+	}
+
+	// check if provider layer are automatically generated
+	var autoTables []string
+	if auto {
+		// get auto config settings
```

Comment:
Should be able to change this if to:

```go
if len(layers) == 0 {
...
```

And then remove all instance of `auto` and `DefaultAuto`